docs/src/modules/components/ApiPage/SlotsList.tsx

```diff
@@ -52,9 +52,12 @@ export default function SlotsList(props: SlotsListProps) {
                 }}
               />
             )}
-            <p className="slot-default-element">
-              <span>{t('api-docs.default')}:</span> <code className="Api-code">{defaultValue}</code>
-            </p>
+            {defaultValue && (
```

Comment:
As far as I know, a slot should always have a default value, I think it's better to have it display broken to make it easer to spot.

Reply:
For Base UI's Slider, developers must need to provide the slot for the value label if they wish to customize it. If they don't provide the slot, the value label won't be displayed. Unlike Material UI or Joy UI, there's no specific prop like `valueLabelDisplay` controlling the value label, giving developers more flexibility in customizing it.

Reply:
Oh I see 👍